docs/UserGuide.md

```diff
@@ -164,34 +168,38 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [e/EMAIL] [g/GENDER] [a/AGE] [t/ADD_TAG]… [-t/DELETE_TAG]…​`
+Format: `edit INDEX [n/NAME] [e/EMAIL] [g/GENDER] [a/AGE] [d/DETAIL] [t/ADD_TAG]… [-t/DELETE_TAG]…`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Any extra duplicate study group tags in the input will be ignored automatically.
+</div>
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The index **must be a positive integer** (e.g., 1, 2, 3, …).
 * At least one of the optional fields must be provided.
 * When editing tags:
   * Prefix:`[t/ADD-TAG]`
-    - the added tags will be **appended** to the list of existing tags i.e adding of tags does not overwrite existing tags.
-    - if the tag name you entered to add **already exists**, the edit will be successful but you will get the warning:
+    - the added tags will be **appended** to the list of existing tags (i.e., adding of tags does not overwrite existing tags).
+    - if the tag name you entered to add **already exists**, the edit will be successful, but you will get the warning:
       `You tried adding an already existing study group tag.`
   * Prefix: `[-t/DELETE-TAG]`
     - the tags specified will be **deleted** from the list of existing tags.
-    - if the tag name you entered to delete **does not exist**, the edit will be successful but you will get the warning:
+    - if the tag name you entered to delete **does not exist**, the edit will be successful, but you will get the warning:
       `You tried removing a nonexistent study group tag.`
 * When editing **all other fields** : Existing values will be updated to the input values.
 
 Examples:
 *  `edit 1 a/29 e/johndoe@example.com` Edits the age and email address of the 1st person to be `29` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/1B -t/2B` Edits the name of the 2nd person to be `Betsy Crower`, adds a new tag `1B` and deletes existing tag `2B`
+*  `edit 2 n/Betsy Crower t/1B -t/2B` Edits the name of the 2nd person to be `Betsy Crower`, adds a new tag `1B` and deletes existing tag `2B`.
 
 ### Deleting persons : `delete`
 
-Deletes the specified persons from the address book.
+Deletes the specified person(s) from the address book.
 
 Format: `delete [INDEX | INDEX-INDEX]…`
 
-* Deletes the person at the specified `INDEX` or range of indices
-* The index must be within the range
```

Comment:
Could you also add a point about requiring at least one optional field?

`* At least one of the index (range) must be provided.`